features/viewing_details_of_credit_memo.feature

```diff
@@ -55,3 +55,11 @@ Feature: Viewing details of a credit memo
         And it should contain 1 "Galaxy Post" shipment with "4.50" gross value in "USD" currency
         And it should be issued in "United States" channel
         And its total should be "4.50" in "USD" currency
+
+    @ui
+    Scenario: Having all credit memos listed on the order page in ascending order
```

Comment:
This file contains several scenarios for viewing details of credit memo, so this added scenario should be extracted to a separate feature file, sth like `seeing_credit_memos_on_admin_order_view.feature` to be consistent with others